### ISSUE TITLE
Explain how to use in Janus

### ISSUE THREAD

**Issue by sirkitree** [timschwartz/janus-method-message#1]
I see this provides the backend functionality, but is there a way to actually utilize this in JanusVR? Can we use the current chat system or do we need James to make some changes to the client in order to utliize this?

**Comment by timschwartz** [comment]
Yes, James will have to implement the command in JanusVR client.

**Comment by sirkitree** [comment]
If that is the case, you'll probably want to post in reddit, otherwise he'll never see it.

**Comment by timschwartz** [comment]
James said he'll add support for this method when beta.vrsites.com supports it.

But he wants some syntax changes first, so no hurry :)
